timemachine/cpp/src/integrator.cu

```diff
@@ -37,9 +37,9 @@ __global__ void update_forward(
     const int N,
     const int D,
     const RealType ca,
-    const RealType *cbs, // N x 3, not P x N x 3, but we could just pass in the first index
-    const RealType *ccs, // N
-    const RealType *noise,
+    const RealType *cbs,   // N x 3, not P x N x 3, but we could just pass in the first index
+    const RealType *ccs,   // N
+    const RealType *noise, // [N x 3]
```

Comment:
Tangent: Should these shape comments not all be N x 3 ?

Reply:
We only access `ccs` using the `atom_idx`. Which makes sense since it contains (based on the comment above its usage) `sqrt(1 - exp(-2 * friction * dt)) * sqrt(kT / mass)` which is independent of the dimensions

Reply:
Ahh, so should `cbs` also be N rather than N x 3? (Also only accessed using `atom_idx`)